tests_oval_graph/test_commands/test_command_arf_to_graph.py

```diff
@@ -94,4 +91,4 @@ def test_command_parameter_all_and_show_failed_rules():
                r'_package_\w+_removed'
                ]
     subprocess.check_call(command)
-    assert len(os.listdir(src)) == 1
+    assert os.path.isfile(src)
```

Comment:
How come that the output of `get_random_dir...` can be a file and not a directory? Perhaps that function should be renamed to refer to `path` instead of `dir`

Next, I would recommend to use `pathlib` everywhere, and then you can write just `src.is_file()` - much shorter!